release documentation [11814] (#82)
eProsima/Fast-DDS-statistics-backend: docs/rst/types/statistic_kind.rst

StatisticKind
=============

> Retrieving data from the *eProsima Fast DDS Statistics Backend* with the |get_data-api|,
> it is possible to specify the kind of statistic we want to receive in the output.

Reviewer: ```suggestion
|get_data-api| allows for retrieving data from the *eProsima Fast DDS Statistics Backend* specifying 
the kind of statistic we want to receive in the output.
```
